app-saml/init.sls

```diff
@@ -0,0 +1,209 @@
+#
```

Comment:
Changed to put all of the configuration related files into a `conf` folder during the state execution and added the relevant scripts to tie into gocd and move the configuration files into the right place :+1: